src/core/plugins/json-schema-5-samples/fn/index.js

```diff
@@ -635,3 +635,7 @@ const resolver = (arg1, arg2, arg3) => [arg1, JSON.stringify(arg2), JSON.stringi
 export const memoizedCreateXMLExample = memoizeN(createXMLExample, resolver)
 
 export const memoizedSampleFromSchema = memoizeN(sampleFromSchema, resolver)
+
+export const getSchemaObjectTypeLabel = (label) => label
+
+export const getSchemaObjectType = (schema) => schema.type ?? "any"
```

Comment:
To be consistent with JSON Schema 2020-12 processing, can we make the fallback type the `string` type?

```suggestion
export const getSchemaObjectType = (schema) => schema.type ?? "string"
```

Also, may I suggest  to make this function safe:


```suggestion
export const getSchemaObjectType = (schema) => schema?.type ?? "string"
```

Reply:
Preferred alternative: What if both the functions would have the following signature?

```
getSchemaObjectType: (schema: object) -> string
getSchemaObjectTypeLabel: (schema: object) -> string
```

This would simplify their usage inside the code as it is now abstracted how the the type or label are computed from the schema.

NOTE: this applies both for Draft 5 and 2020-12 functions.